exchange/exchange.go

```diff
@@ -979,10 +977,22 @@ func makeBidExtJSON(ext json.RawMessage, prebid *openrtb_ext.ExtBidPrebid, impEx
 		extMap = make(map[string]interface{})
 	}
 
+	// ext.prebid
+	if prebid.Meta == nil && maputil.HasElement(extMap, "prebid", "meta") {
```

Comment:
My idea was to reuse existing `jsonutil`, but I'm ok with this too :)

Reply:
In `fpd` branch I modified `jsonutil` to support exactly what you wrote but with byte array:
`func FindElement(extension []byte, elementNames ...string)` - to check element. with any nexting level exists
and 
`func SetElement(originDataInput []byte, setValue []byte, keys ...string)` - to set element with any nested level.
Using this functions will replace logic related to `extMap` variable and Unmarshal and Marshal will not be necessary anymore

Reply:
Thank you for sharing. The `jsontuil` methods operate on a json byte array and new `maputil` is operating on an already unmashalled map. There are other functions in this method which require full blown json marshal. So, I don't think the `jsontuil` would be a good use in this case.